Description: use new /usr/share/sgml location for docbook.dsl
Author: Matthias Klose <doko@debian.org>
Forwarded: no
Last-Update: 2013-10-30

--- a/Docs/sgmlmanual/Makefile
+++ b/Docs/sgmlmanual/Makefile
@@ -88,14 +88,14 @@
 	$(MAKE) dirimages htmlimages
 	rm -f $@
 	jade -t rtf -o $*.rtf.tmp \
-		-d /usr/lib/sgml/stylesheet/dsssl/docbook/nwalsh/print/docbook.dsl $<
+		-d /usr/share/sgml/docbook/stylesheet/dsssl/modular/print/docbook.dsl $<
 	cat $*.rtf.tmp | sed 's,"img/\(.*\)\.[^.]*","img.tmp/\1.gif",g' >$@
 	rm -f $*.rtf.tmp
 
 %.tex: %.sgml
 	rm -f $@
 	jade -t tex -o $*.tex.tmp \
-		-d /usr/lib/sgml/stylesheet/dsssl/docbook/nwalsh/print/docbook.dsl $<
+		-d /usr/share/sgml/docbook/stylesheet/dsssl/modular/print/docbook.dsl $<
 	cat $*.tex.tmp | sed 's,{img/\(.*\)\.[^.]*},{img.tmp/\1.eps},g' >$@
 	rm -f $*.tex.tmp
 
